frontend/app/containers/App/constants.js

```diff
@@ -9,7 +9,12 @@
  * export const YOUR_ACTION_CONSTANT = 'yourproject/YourContainer/YOUR_ACTION_CONSTANT';
  */
 
-export const LOAD_REPOS = 'boilerplate/App/LOAD_REPOS';
-export const LOAD_REPOS_SUCCESS = 'boilerplate/App/LOAD_REPOS_SUCCESS';
-export const LOAD_REPOS_ERROR = 'boilerplate/App/LOAD_REPOS_ERROR';
+export const LOAD_NAMESPACES = 'boilerplate/App/LOAD_NAMESPACES';
```

Comment:
Can we rename the project from `boilerplate` to `bebaios`?
```suggestion
export const LOAD_NAMESPACES = 'bebaios/App/LOAD_NAMESPACES';
```